src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -46,54 +46,29 @@
 
 public abstract class EasyPostResource {
     public static final String EASYPOST_SUPPORT_EMAIL = "support@easypost.com";
-
-    private static final int DEFAULT_CONNECT_TIMEOUT_MILLISECONDS = 30000;
-    private static final int DEFAULT_READ_TIMEOUT_MILLISECONDS = 60000;
-    private static final double APP_ENGINE_DEFAULT_TIMEOUT_SECONDS = 20.0;
-
     public static final Gson GSON =
             new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
                     .registerTypeAdapter(Event.class, new EventDeserializer())
                     .registerTypeAdapter(Rate.class, new RateDeserializer())
                     .registerTypeAdapter(SmartrateCollection.class, new SmartrateCollectionDeserializer()).create();
-
     public static final Gson PRETTY_PRINT_GSON = new GsonBuilder().setPrettyPrinting().serializeNulls()
             .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
             .registerTypeAdapter(Event.class, new EventDeserializer()).create();
-
-    /**
-     * Returns a string representation of the object.
-     */
-    @Override
-    public String toString() {
-
-        return (String) this.getIdString();
-    }
-
-    /**
-     * Pretty print the JSON representation of the object.
-     *
-     * @return the JSON representation of the object.
-     */
-    public String prettyPrint() {
-        return String.format("<%s@%s id=%s> JSON: %s", this.getClass().getName(), System.identityHashCode(this),
-                this.getIdString(), PRETTY_PRINT_GSON.toJson(this));
-    }
-
-    private Object getIdString() {
-        try {
-            Field idField = this.getClass().getDeclaredField("id");
-            return idField.get(this);
-        } catch (SecurityException e) {
-            return "";
-        } catch (NoSuchFieldException e) {
-            return "";
-        } catch (IllegalArgumentException e) {
-            return "";
-        } catch (IllegalAccessException e) {
-            return "";
-        }
-    }
+    public static final String CHARSET = "UTF-8";
+    public static final ArrayList<String> GLOBAL_FIELD_ACCESSORS =
+            new ArrayList<>(Arrays.asList("getCreatedAt", "getUpdatedAt", "getFees"));
+    private static final int DEFAULT_CONNECT_TIMEOUT_MILLISECONDS = 30000;
+    private static final int DEFAULT_READ_TIMEOUT_MILLISECONDS = 60000;
+    private static final double DEFAULT_APP_ENGINE_TIMEOUT_SECONDS = 20.0;
+    private static final String DNS_CACHE_TTL_PROPERTY_NAME = "networkaddress.cache.ttl";
+    // Set this property to override your environment's default URLStreamHandler.
+    private static final String CUSTOM_URL_STREAM_HANDLER_PROPERTY_NAME = "com.easypost.net.customURLStreamHandler";
```

Comment:
How would someone be able to set this property if it is declared as a `final` ?

Reply:
This property doesn't have a setter anywhere, and since it's private, wouldn't be able to be accessed directly.
Should we remove the comment or add in a way to set this?

Reply:
If we expect users would need/want to edit this, we should probably add getters/setters. If not, we could remove the comment. I frankly don't know off-hand what this is used for.

Reply:
This has been a private variable with no setter/getter from the very beginning of the library (e0e912c2233d1720b065438e435ab6d4adcac93d). So, whatever it is, it's always been this way.
I vote for removing the comment.

Reply:
Let's remove the comment. If someone really needs this bit we can always tweak it later.

Reply:
Resolved in 4abf9d9